modules/mobile/app/serializers/mobile/v0/debts_serializer.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class DebtsSerializer
+      include JSONAPI::Serializer
+
+      set_type :debts
+
+      attributes :fileNumber,
+                 :payeeNumber,
+                 :personEntitled,
+                 :deductionCode,
+                 :benefitType,
+                 :diaryCode,
+                 :diaryCodeDescription,
+                 :amountOverpaid,
+                 :amountWithheld,
+                 :originalAR,
+                 :currentAR,
+                 :debtHistory
+      def initialize(debts, id = nil)
+        resource = debts.map do |debt|
+          DebtStruct.new(id: id || debt['id'],
+                         fileNumber: debt['fileNumber'],
+                         payeeNumber: debt['payeeNumber'],
+                         personEntitled: debt['personEntitled'],
+                         deductionCode: debt['deductionCode'],
+                         benefitType: debt['benefitType'],
+                         diaryCode: debt['diaryCode'],
+                         diaryCodeDescription: debt['diaryCodeDescription'],
+                         amountOverpaid: debt['amountOverpaid'],
+                         amountWithheld: debt['amountWithheld'],
+                         originalAR: debt['originalAR'],
+                         currentAR: debt['currentAR'],
+                         debtHistory: debt['debtHistory'])
+        end
+
+        resource = resource.first if resource.size == 1
```

Comment:
If this is an index fetch that returns a single record, won't this break the schema by not returning an array? I think there are a few ways we can fix this. One might be to remove the array.wrap from the controller. Then just do something like:
```
 resource = if debts.is_a? Array
  debts.map(serialize_debt(debt))
else
  serialize_debt(debt)
end

def serialize_debt(debt)
   DebtStruct.new(debt['id'], etc.)
end
```

It's a little awkward with the naming because this is the debts serializer and the input is debts. There are other options though.

Reply:
yeah, good call. updated to use that logic